metricbeat/docs/modules/golang.asciidoc

```diff
@@ -0,0 +1,42 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-module-golang]]
+== golang Module
+
+This is the golang Module. Metrics are collected submitting HTTP GET requests to golang-expvar-api.
+Reference: https://golang.org/pkg/expvar/
+
+[float]
+=== Example Configuration
+
+The golang module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+#- module: golang
+#  metricsets: ["expvar","heap"]
+#  enabled: true
+#  period: 10s
+#  hosts: ["localhost:6060"]
+#  expvar.path: "/debug/vars"
+#  heap.path: "/debug/vars"
+#  expvar.namespace: "example"
```

Comment:
would be good to group `expvar` config options together. Can be done in a follow up PR if you like.